src/Client/ClickHouseAsyncClient.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SimPod\ClickHouseClient\Client;
+
+use GuzzleHttp\Promise\PromiseInterface;
+use SimPod\ClickHouseClient\Format\Format;
+use SimPod\ClickHouseClient\Output\Output;
+
+interface ClickHouseAsyncClient
+{
+    /**
+     * @see Output hack for IDE to preserve `use`
```

Comment:
tenhle komentář nechápu...

Reply:
IDE je dumb a nepozna usage u `* @phpstan-template O of Output` takze mi ho chce vymazat pri Optimize imports. takhle forcnu usage

Reply:
docela na hovno takový hacky, je to hrozně matoucí :D

Reply:
jn, ale issue jetbrains ma. doufam ze 2020.1 se hecnou, to php zaspali